log/aggregator.go

```diff
@@ -0,0 +1,18 @@
+package log
+
+import (
+	"time"
+)
+
+// Aggregator is an interface for pluggable components that collect log messages delivered via
+// some transport mechanism
+type Aggregator interface {
```

Comment:
I'm really bad at naming too, so I'm ok with going with this. Also, I saw that you also merged/moved some packages around after I wrote the OP, so I'm now 2x ok with this 😄